tidb-cloud/setup-azure-self-hosted-kafka-private-link-service.md

```diff
@@ -0,0 +1,544 @@
+
+    1. In the [TiDB Cloud console](https://tidbcloud.com), navigate to the [**Clusters**](https://tidbcloud.com/console/clusters) page, and then click the name of your target cluster to go to its overview page.
+    2. In the left navigation pane, click **Changefeed**.
+    3. On the **Changefeed** page, click **Create Changefeed** in the upper-right corner.
+        1. In **Destination**, select **Kafka**.
+        2. In **Connectivity Method**, select **Private Link**.
+    4. Note down the region information and the subscription of the TiDB Cloud Azure account in **Reminders before proceeding**. You will use it to authorize TiDB Cloud to access the Kafka Private Link service.
+    5. Enter a unique **Kafka Advertised Listener Pattern** for your Kafka Private Link service.
+        1. Input a unique random string. It can only include numbers or lowercase letters. You will use it to generate **Kafka Advertised Listener Pattern** later.
+        2. Click **Check usage and generate** to check if the random string is unique and generate **Kafka Advertised Listener Pattern** that will be used to assemble the EXTERNAL advertised listener for Kafka brokers. 
+
+Note down all the deployment information. You need to use it to configure your Kafka Private Link service later.
+
+The following table shows an example of the deployment information.
+
+| Information     | Value    | Note    | 
+|--------|-----------------|---------------------------|
+| Region | Virginia (`eastus`) | N/A |
+| Subscription of TiDB Cloud Azure account | `99549169-6cee-4263-8491-924a3011ee31` | N/A |
+| Kafka Advertised Listener Pattern | The unique random string: `abc` | Generated pattern: `<broker_id>.abc.eastus.azure.3199745.tidbcloud.com:<port>`; |
+
+## Step 1. Set up a Kafka cluster
+
+If you need to deploy a new cluster, follow the instructions in [Deploy a new Kafka cluster](#deploy-a-new-kafka-cluster).
+
+If you need to expose an existing cluster, follow the instructions in [Reconfigure a running Kafka cluster](#reconfigure-a-running-kafka-cluster). 
+
+### Deploy a new Kafka cluster
+
+#### 1. Set up the Kafka virtual network
+
+1. Log in to the [Azure portal](https://portal.azure.com/), go to the [Virtual networks](https://portal.azure.com/#browse/Microsoft.Network%2FvirtualNetworks) page, and then click **+ Create** to create a virtual network.
+2. In the **Basic** tab, select your **Subscription**, **Resource group**, and **Region**, enter a name (for example, `kafka-pls-vnet`) in the **Virtual network name** field, and then click **Next**.
+3. In the **Security** tab, enable Azure Bastion, and then click **Next**.
+4. In the **IP addresses** tab, do the following:
+
+    1. Set the address space of your virtual network, for example, `10.0.0.0/16`.
+    2. Click **Add a subnet** to create a subnet for brokers, fill in the following information, and then click **Add**.
+        - **Name**: `brokers-subnet`
+        - **IP address range**: `10.0.0.0 - 10.0.0.255`
```

Comment:
In "IP address range", if we choose `10.0.0.0/16`, the corresponding value is `10.0.0.0 - 10.0.255.255`.
![image](https://github.com/user-attachments/assets/aec71cbe-7d78-468e-9838-401226e26104)